### PR TITLE
Command Key (MetaKey) on MacOS as default

### PR DESCRIPTION
**Pull request by lifeeric**
I came across this project and I found it really amazing. but when I visited it and wrote a simple prompt and hit "CMD + ENTER" on MacOS, it didn't work. however with "CTR + ENTER" it worked. On MacOS it's really easy to press the "command" key instead of CTR.

I created a PR that fixes this issue, now the default key submitKey should be MetaKey on MacOS.

Hope it helps the project & community!

**DEMO**: https://nextgpteric.vercel.app/

### CODE REVIEW

File: app/utils.ts [hunk 1]
```diff
@@ -178,10 +178,10 @@ export function getCSSVar(varName: string) {
  * Detects if the Operation system is MacOS
  */
 export function isMacOS(): boolean {
-    if (window !== 'undefined') {
-        let userAgent = window?.navigator?.userAgent
-        if (userAgent.indexOf('Mac') != -1) return true
-    }
+  if (typeof window !== "undefined") {
+    let userAgent = window?.navigator?.userAgent;
+    if (userAgent.indexOf("Mac") != -1) return true;
```
Comment (H0llyW00dzZ) [inline app/utils.ts:183]: must refactor this instead of `Mac` its should be `macintosh` with parse lowerstring

Comment (H0llyW00dzZ) [reply app/utils.ts:183]: recommended ref code snippet : 
```ts
export function isMacOS(): boolean {
  const userAgent = navigator.userAgent.toLowerCase();
  const isMac = /(iphone|ipad|ipod|macintosh)/i.test(userAgent);
  const isTouchDevice = userAgent.includes("mac") && "ontouchend" in document;
  
  return isMac && isTouchDevice;
}
```

Comment (lifeeric) [reply app/utils.ts:183]: @H0llyW00dzZ  Yooo. thank you so much for reviewing it! it helps me improve, that's very kind of you & I really love it. Honestly, you're a legend!


I'm pretty sure, we don't need to detect touchend events, right?

Comment (H0llyW00dzZ) [reply app/utils.ts:183]: @lifeeric  other mac like macbook pro with touch bar

Comment (lifeeric) [reply app/utils.ts:183]: @H0llyW00dzZ If we already found out that it is a macintosh. we still need `ontouched` event since we don't use it?

Thank you

Comment (H0llyW00dzZ) [reply app/utils.ts:183]: @lifeeric Just in case, for the benefit of others, not just us.

Comment (lifeeric) [reply app/utils.ts]: @H0llyW00dzZ please have a look again. thanks!

**Comment by H0llyW00dzZ** [comment]
nice 
Added to my forks

image

**Comment by lifeeric** [comment]
@H0llyW00dzZ Thank you so much & You're the best!

I also learned from you: signing commit with GPG.

**Pull request merged**